sub/ass_mp.c

```diff
@@ -123,10 +124,24 @@ static const int map_ass_level[] = {
 
 static void message_callback(int level, const char *format, va_list va, void *ctx)
 {
+    static atomic_bool supress_missing_glyph_warning;
     struct mp_log *log = ctx;
     if (!log)
         return;
     level = map_ass_level[level];
+    // "fontselect" missing glyph messages are treated as "warnings" by libass.
+    // However, they're highly disruptive as these messages can be genereated
+    // repeatedly every time the sub is rendered, creating log spam.
+    // Unfortunately, libass also has some other useful messages at the same
+    // loglevel, so use strncmp to check the presence of fontselect messages.
+    // Reduce the log spam by changing the log level after the first warning.
+    if (level == MSGL_INFO && strncmp(format, "fontselect: failed to find", 26) == 0) {
+        if (atomic_load(&supress_missing_glyph_warning)) {
+            level = MSGL_V;
+        } else {
+            atomic_store(&supress_missing_glyph_warning, true);
+        }
```

Comment:
for simple types, no need to go through `atomic_load/store`

also to avoid atomic store on each message, I would reverse the logic (also need change in other part)
```suggestion
        if (!missing_glyph_warned)) {
            level = MSGL_INFO;
            missing_glyph_warned = true;
        } else {
            level = MSGL_V;
        }
```

Also in fact I think we can even not use atomic, even if we race here, nothing bad will happen. But it is fine too as is.

Reply:
Forgot to remove the stdatomic include